specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2024-05-15-preview/networkSecurityPerimeter.json

```diff
@@ -0,0 +1,455 @@
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfigurationList": {
+      "description": "Result of the List Network Security Perimeter configuration operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A collection of Network Security Perimeter configurations",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkSecurityPerimeterConfiguration"
+          }
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfigurationProperties": {
+      "description": "Properties of the Network Security Perimeter Configuration",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning state of Network Security Perimeter configuration propagation",
+          "enum": [
+            "Accepted",
+            "Succeeded",
+            "Failed",
+            "Deleting",
+            "Canceled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "NetworkSecurityPerimeterConfigurationProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "provisioningIssues": {
+          "description": "List of Provisioning Issues if any",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ProvisioningIssue"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "networkSecurityPerimeter": {
+          "$ref": "#/definitions/NetworkSecurityPerimeter",
+          "readOnly": true
+        },
+        "resourceAssociation": {
+          "description": "Information about resource association",
+          "readOnly": true,
+          "type": "object",
+          "properties": {
+            "name": {
+              "description": "Name of the resource association",
+              "type": "string"
+            },
+            "accessMode": {
+              "description": "Access Mode of the resource association",
+              "enum": [
+                "Enforced",
+                "Learning",
+                "Audit"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "ResourceAssociationAccessMode",
+                "modelAsString": true
+              }
+            }
+          }
+        },
+        "profile": {
+          "description": "Network Security Perimeter profile",
+          "readOnly": true,
+          "type": "object",
+          "properties": {
+            "name": {
+              "description": "Name of the resource",
```

Comment:
>resource [](http://example.com/codeflow?start=42&length=8)

'Name of the profile' would be clearer

Reply:
This is inside the json of the profile so Name of the profile